Pathogens/Arbo/ETL/constants.py

```diff
@@ -25,10 +25,15 @@
         # 'source_sheet',
         # 'source_sheet_id',
         'state',
-        'url'
+        'url',
+        'who_region',
+        'estimate_id',
+        'source_sheet_name'
```

Comment:
so source sheet is supposed to be the name: https://airtable.com/appeuLEYGpJjE6PHF/tblHgbUz3yFanlHmF/viwTHvxURbphQthRG?blocks=hide

I believe this is a join on the airtable side too which is why it extracted the ids. 

Also note on the 2 seperate table structure we have. Other than avoiding minro duplication idk if there is value in having 2 seperate tables atleast at the moment. I think if we ever added a feature to let people access source sheet level data it could be something we did, but I thought it would be worth it to keep things simple for now and have it all in 1 table. lmk your thoughts on this as I do not have much DB architecture experience and am just doing what seems write based on shit I read online

The issue rn is that we would have to access both tables in airtable. Since we have Study/Estimate sheet and the the Source Sheet which are both seperate but linked in airtable as well. Estimate id represents the id of each estimate study inside a source sheet. It distinguishes each estimate in a source sheet from the other as well as from all other estimates (is a Unique Id). I was considering showing it in the app since it is the only unique identifier that is also very descriptive. Do not want to clean it up without checking with data team as I feel there is value in keeping it the way it was constructed for ease of cross comparison.  

@harrietware thoughts on cleaning up estimate_id and displaying it/not displaying it?

Reply:
Hey Himanshu,

I've opened a PR based on our discussion earlier that should do the trick for us. Feel free to take a look when you've got a spare moment.

https://github.com/serotracker/iit-backend/pull/336

I've tested the ETL with my changes and am fully on board with running it in prod so if you are too let's :shipit: along with the sample size and latlng stuff :)

Reply:
As discussed, please add estimate_id as the leftmost column in the data table

Reply:
Front end work added to ticket https://github.com/PathoTracker/Pathotracker/issues/47